### ISSUE TITLE
Project templates should set WinUISDKReferences=false

### ISSUE THREAD

**Issue by Scottj1s** [microsoft/WindowsAppSDK#2425]
### Describe the bug

With v1.0.3 and v1.1P2, WinAppSDK has fully adopted Hybrid CRT, and a VCLibs.UPWDesktop framework package reference is no longer required.  The WinUI appx targets file automatically adds this reference for backwards compat, but it can be removed for new projects.

### Steps to reproduce the bug

Project templates should include this property:

`<WinUISDKReferences>false</WinUISDKReferences>`

**Comment by jkarppila** [comment]
Unfortunately setting WinUISDKReferences to false causes applications utilizing Win2D to crash (using appsdk 1.1 preview 2, Win2D 1.0.1). Manually installing the redistributables old fashioned way (downloading and running VC_redist.x64.exe) to machine fixes this, but is quite inconvenient.

Could this be fixed with some additional settings? Or does it require new Win2D version?

**Comment by Scottj1s** [comment]
> Could this be fixed with some additional settings? Or does it require new Win2D version?

Yes, this requires a new version of Win2D, which is currently also being converted to Hybrid CRT.

**Comment by riverar** [comment]
> With v1.0.3 and v1.1P2, WinAppSDK has fully adopted Hybrid CRT

So this is no longer accurate, right? Not trying to ding you here but was announced far and wide (e.g. on the WinUI community call) and I just want to have the correct facts on hand.

**Comment by jkarppila** [comment]
> Yes, this requires a new version of Win2D, which is currently also being converted to Hybrid CRT.

Thanks. Issue 874 in Win2D repo seems to address this at some point.

**Comment by Marv51** [comment]
Can I ask what the expected outcome of the WinUISDKReferences property is? 

I was expecting a removal of the dependencies: `Microsoft.VCLibs.140.00.UWPDesktop` and `Microsoft.VCLibs.140.00`

However, when I try it on my project it also removes the output of the dependency `Microsoft.WindowsAppRuntime.1.0` which at least in 1.0.x should not be possible, right? I don't think the package created with this property is run able.

Can you clarify this for me? @Scottj1s
